docs/pages/docs/api-reference/generative-ui/create-ai.mdx

```diff
@@ -10,21 +10,24 @@ import { Tabs, Tab } from 'nextra-theme-docs';
 
 ## `createAI`
 
-`createAI` is a function that creates a new `ai/rsc` instance.
+`createAI` creates a new `ai/rsc` instance.
 
 ## Parameters
 
-### `AIState`
+### `initialAIState`: [`AIState`](/docs/concepts/ai-rsc#aistate)
 
-`AIState` is a JSON representation of all the context the LLM needs to read. Usually for a chat app, `AIState` contains the textual conversation history between the user and the assistant. In practice, it can also be used to store other values and meta information such as `createdAt` of each message. `AIState` by default, can be accessed/modified on both Server and Client.
+`AIState` is a JSON representation of all the context the LLM needs to read. Usually for a chat app, `AIState` contains the textual conversation history between the user and the assistant. It can also be used to store other values and information such as a `createdAt` field for each message.
```

Comment:
```suggestion
`AIState` is a JSON representation of all the context the LLM needs to read. Usually for a chat app, `AIState` contains the textual conversation history between the user and the assistant. It can also be used to store other values and information such as a `createdAt` field for each message, or a `chatId` field besides all the messages.
```